src/components/view-switcher/ViewSwitcher.tsx

```diff
@@ -1,32 +1,41 @@
+import { FC, MouseEvent } from 'react'
+
 import ToggleButton from '@mui/material/ToggleButton'
 import Box from '@mui/material/Box'
 
 import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted'
 import GridViewIcon from '@mui/icons-material/GridView'
 
-import { cardsViews } from '~/constants'
 import { styles } from '~/components/view-switcher/ViewSwitcher.styles'
 
-const ViewSwitcher = ({ setOffersView, offersView }) => {
-  const changeOffersView = (_event, view) => setOffersView(view)
+import { CardsViewEnums } from '~/types/findOffers/enums/viewSwitcher.enums'
```

Comment:
Now you had better to import all type declarations from '~types', look at structure and other files

Reply:
changed imports of types